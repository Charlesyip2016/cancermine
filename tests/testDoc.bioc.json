[
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "The HOX1-MLL2 fusion gene is a tumor suppressor in colorectal cancer."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "The HER2-neu gene is a tumor suppressor in skin cancer."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "The HER2/neu gene is a tumor suppressor in glioblastoma."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "mir-1 is a tumor suppressor in lung cancer."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "mir-5 is a tumor suppressor in non-small-cell lung carcinoma (NSCLC)."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "HER2 (ERBB2) is a tumor suppressor in pancreatic cancer."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "EGFR is a tumor suppressor in null cell acute lymphoblastic leukaemia (NCALL)."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "mir1 is an oncogene in leukemia."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "mir100 is an oncogene in lymphoma."
  },
  {
    "title": "A fun title for a paper",
    "pmid": "12345678",
    "pmcid": "110011001100",
    "year": "2000",
    "journal": "Test Journal",
    "journalISO": "T. Journal",
    "section": "abstract",
    "id": "28330089",
    "sentence": "DPP9-AS1 drives acute lymphoblastic leukemia."
  }
]