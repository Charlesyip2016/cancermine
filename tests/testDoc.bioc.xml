<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<collection>
  <source/>
  <date>2017-12-06</date>
  <key/>
  <document>
    <id>28330089</id>
    <infon key="title">A fun title for a paper</infon>
    <infon key="pmid">12345678</infon>
    <infon key="pmcid">110011001100</infon>
    <infon key="year">2000</infon>
    <infon key="journal">Test Journal</infon>
    <infon key="journalISO">T. Journal</infon>
    <passage>
      <offset>0</offset>
      <text>
      The HOX1-MLL2 fusion gene is a tumor suppressor in colorectal cancer.
      The HER2-neu gene is a tumor suppressor in skin cancer.
      The HER2/neu gene is a tumor suppressor in glioblastoma.
      mir-1 is a tumor suppressor in lung cancer.
      mir-5 is a tumor suppressor in non-small-cell lung carcinoma (NSCLC).
      HER2 (ERBB2) is a tumor suppressor in pancreatic cancer.
      EGFR is a tumor suppressor in null cell acute lymphoblastic leukaemia (NCALL).
      </text>
      <infon key="section">abstract</infon>
    </passage>
  </document>
</collection>
