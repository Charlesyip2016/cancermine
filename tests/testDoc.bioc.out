12345678	A fun title for a paper	Test Journal	2000	abstract	Tumor_Suppressor	DOID:5672;DOID:9256	colorectal cancer	colorectal cancer	combo|HGNC:5099;HGNC:5100;HGNC:5101;HGNC:5102;HGNC:5104;HGNC:5105;HGNC:5106;HGNC:5107;HGNC:5108;HGNC:5109|HGNC:15840;HGNC:7133	HOX1-MLL2	HOXA1;HOXA10;HOXA11;HOXA13;HOXA3;HOXA4;HOXA5;HOXA6;HOXA7;HOXA9|KMT2B;KMT2D	
      The HOX1-MLL2 fusion gene is a tumor suppressor in colorectal cancer.
      
12345678	A fun title for a paper	Test Journal	2000	abstract	Tumor_Suppressor	DOID:4159	skin cancer	skin cancer	HGNC:3430	HER2-neu	ERBB2	The HER2-neu gene is a tumor suppressor in skin cancer.
      
12345678	A fun title for a paper	Test Journal	2000	abstract	Tumor_Suppressor	DOID:3068	glioblastoma	glioblastoma multiforme	HGNC:3430	HER2/neu	ERBB2	The HER2/neu gene is a tumor suppressor in glioblastoma.
      
12345678	A fun title for a paper	Test Journal	2000	abstract	Tumor_Suppressor	DOID:1324;DOID:3905	lung cancer	lung cancer	mirna|mir-1	mir-1	miR-1	mir-1 is a tumor suppressor in lung cancer.
      
12345678	A fun title for a paper	Test Journal	2000	abstract	Tumor_Suppressor	DOID:3908	non-small-cell lung carcinoma (NSCLC)	non-small cell lung carcinoma	mirna|mir-5	mir-5	miR-5	mir-5 is a tumor suppressor in non-small-cell lung carcinoma (NSCLC).
      
12345678	A fun title for a paper	Test Journal	2000	abstract	Tumor_Suppressor	DOID:1793;DOID:4905	pancreatic cancer	pancreatic cancer	HGNC:3430	HER2 (ERBB2)	ERBB2	HER2 (ERBB2) is a tumor suppressor in pancreatic cancer.
      
12345678	A fun title for a paper	Test Journal	2000	abstract	Tumor_Suppressor	DOID:9954	null cell acute lymphoblastic leukaemia	null-cell leukemia	HGNC:3236	EGFR	EGFR	EGFR is a tumor suppressor in null cell acute lymphoblastic leukaemia (NCALL).
      
